.travis.yml

```diff
@@ -2,7 +2,8 @@ language: go
 
 go:
   - "1.10"
-  - "tip"
+  - "1.11"
```

Comment:
```suggestion
  - "1.11"
```

Testing just 1.10 and 1.12 seems enough to me

Reply:
Done (d95fe3a4).

Reply:
(I folded this change in with 49d1a65 as part of #4)